LibreNMS/__init__.py

```diff
@@ -0,0 +1,244 @@
+import threading
+from logging import critical
+from time import time
+
+from .service import Service, ServiceConfig
+from .queuemanager import QueueManager, TimedQueueManager, BillingQueueManager
+
+
+def normalize_wait(seconds):
+    return seconds - (time() % seconds)
+
+
+class DB:
+    def __init__(self, config, auto_connect=True):
+        """
+        Simple DB wrapper
+        :param config: The poller config object
+        """
+        self.config = config
+        self._db = None
+
+        if auto_connect:
+            self.connect()
+
+    def connect(self):
+        try:
+            import MySQLdb
```

Comment:
Replaced with this to avoid having to install the MySQL developer kit and a C compiler:

```
        try:
            import pymysql
            MySQLdb = pymysql
        except ImportError:
            try:
                import MySQLdb

            except ImportError:
                print("ERROR: missing the mysql python module:")
                print("Either through your os software repository or 'pip install mysqlclient'")
                raise
```